sdk/test/Services/QueryCompatible/Generated/Model/Internal/MarshallTransformations/CreateQueueResponseUnmarshaller.cs

```diff
@@ -0,0 +1,135 @@
+        {
+            CreateQueueResponse response = new CreateQueueResponse();
+
+            context.Read();
+            int targetDepth = context.CurrentDepth;
+            while (context.ReadAtDepth(targetDepth))
+            {
+                if (context.TestExpression("QueueUrl", targetDepth))
+                {
+                    var unmarshaller = StringUnmarshaller.Instance;
+                    response.QueueUrl = unmarshaller.Unmarshall(context);
+                    continue;
+                }
+            }
+
+            return response;
+        }
+
+        /// <summary>
+        /// Unmarshaller error response to exception.
+        /// </summary>  
+        /// <param name="context"></param>
+        /// <param name="innerException"></param>
+        /// <param name="statusCode"></param>
+        /// <returns></returns>
+        public override AmazonServiceException UnmarshallException(JsonUnmarshallerContext context, Exception innerException, HttpStatusCode statusCode)
+        {
+            var errorResponse = JsonErrorResponseUnmarshaller.GetInstance().Unmarshall(context);
+            errorResponse.InnerException = innerException;
+            errorResponse.StatusCode = statusCode;
+
+            var responseBodyBytes = context.GetResponseBodyBytes();
+
+            using (var streamCopy = new MemoryStream(responseBodyBytes))
+  
+            using (var contextCopy = new JsonUnmarshallerContext(streamCopy, true, context.ResponseData))
+            {
+                if (errorResponse.Code != null && errorResponse.Code.Equals("QueueDeletedRecently"))
+                {
+                    return QueueDeletedRecentlyExceptionUnmarshaller.Instance.Unmarshall(contextCopy, errorResponse);
+                }
+                if (errorResponse.Code != null && errorResponse.Code.Equals("QueueAlreadyExists"))
+                {
+                    return QueueNameExistsExceptionUnmarshaller.Instance.Unmarshall(contextCopy, errorResponse);
+                }
+            }
+            var errorCode = errorResponse.Code;
+            var errorType = errorResponse.Type;
+            var queryHeaderKey = Amazon.Util.HeaderKeys.XAmzQueryError;
+            if (context.ResponseData.IsHeaderPresent(queryHeaderKey))
+            {
+                var queryError = context.ResponseData.GetHeaderValue(queryHeaderKey);
+                if (!string.IsNullOrEmpty(queryError) && queryError.Contains(";"))
+                {
+                    var queryErrorParts = queryError.Split(';');                    
+                    if (queryErrorParts.Length == 2)
+                    {
+                        errorCode = queryErrorParts[0];
+                        var errorTypeString = queryErrorParts[1];
+                        if (Enum.IsDefined(typeof(ErrorType), errorTypeString))
+                        {
+                            errorType = (ErrorType)Enum.Parse(typeof(ErrorType), errorTypeString);
```

Comment:
This is subtle, but the generator outputs this line with an extra space:

```suggestion
                            errorType = (ErrorType) Enum.Parse(typeof(ErrorType), errorTypeString);
```

Our full build fails when it detects generated changes that weren't explicitly committed to prevent us from releasing something accidentally.